networking/eth2/src/main/java/tech/pegasys/artemis/networking/eth2/ActiveEth2Network.java

```diff
@@ -125,11 +131,21 @@ public BeaconChainMethods getBeaconChainMethods() {
 
   @Override
   public void subscribeToAttestationCommitteeTopic(final int committeeIndex) {
+    if (aggregateGossipManager == null) {
+      LOG.warn(
+          "Attestation committee can not be subscribed due to gossip manager not being initialized");
+      return;
+    }
```

Comment:
This still worries me.  I suspect we should be storing a list of topics to subscribe to when we do startup or at least throwing an exception to let the caller know the subscription failed rather than ignoring the request.  At the moment it's only called from the `ValidatorApiHandler` so throwing an exception probably makes sense so the validator knows it's subscription failed.  It shouldn't hit it at the moment because of the way it manages timing.

Reply:
Makes sense. Done.